### PR TITLE
Feature/add story flow

### PR DESCRIPTION
**Pull request by rajanand02**
Added the following workflows
1. work on stories assigned to the user

mystories

2. work on unassigned stories

my stories

3. fuzzy search

fuzzy search

Added the following binaries

1. `pivotal-flow`
2. `pf`

Added editorconfig, prettier, eslint, lint-staged and husky

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: http://EditorConfig.org
```
Comment (rheaditi) [inline .editorconfig]: 🎉

File: index.js [hunk 2]
```diff
@@ -59,7 +65,7 @@ const confirmSetup = () => {
 // axios basic config
 const request = axios.create({
   baseURL: `https://www.pivotaltracker.com/services/v5`,
-  timeout: 3000,
+  timeout: 10000, // search could be really slow in pivotal
```
Comment (rheaditi) [inline index.js]: 😟

File: package.json [hunk 3]
```diff
@@ -4,9 +4,11 @@
   "description": "Automate your pivotal workflow",
   "main": "index.js",
   "bin": {
+    "pivotal-flow": "index.js",
     "pivotal-flow-start": "index.js",
```
Comment (rheaditi) [inline package.json]: should we use `pivotal-flow start` `pivotal-flow check` ? also `pf` & `pfs` seem like they should be local aliases (after we saw how npx works) - seems risky.

**Comment by rheaditi** [comment]
Can we expand the story details after it's selected & before the question for start a branch?
To show the description & points etc..

File: index.js (hunk 2 above)
Comment (rajanand02) [reply index.js]: pivotal search responses are very slow

**Comment by rajanand02** [comment]
> Can we expand the story details after it's selected & before the question for start a branch?
> To show the description & points etc..

Yes, Nice idea. We already have all the story information.

File: package.json (hunk 3 above)
Comment (rajanand02) [reply package.json]: ideally yes. now we have to make changes to read input. Can you raise an another PR over the weekend with the mentioned changes?

**Comment by rajanand02** [comment]
> Can we expand the story details after it's selected & before the question for start a branch?
> To show the description & points etc..

@rheaditi addressed this :point_up:

**Comment by rajanand02** [comment]
> Can we expand the story details after it's selected & before the question for start a branch?
> To show the description & points etc..

image

File: package.json (hunk 3 above)
Comment (rheaditi) [reply package.json]: Sure! :+1:

**Pull request merged**